### PR TITLE
Copyright headers update from EPL v1 to EPL v2

### PR DESCRIPTION
**Pull request by jakub-pomykala**
- Added in Contributing.md and LICENSE files
- Updated all the pre-existing copyright headers to EPL v2.0
- Added in copyright headers to the source files that were missing them

### CODE REVIEW

**Review by jacobwdv — APPROVED** [review]
LGTM, thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributing to Open Liberty guides
```
Comment (jimmy1wu) [inline CONTRIBUTING.md:1]: this file prob needs to be updated to reflect the space rover instead of the guides

Comment (jakub-pomykala) [reply CONTRIBUTING.md]: Done

Comment (jakub-pomykala) [reply CONTRIBUTING.md]: @jimmy1wu would you be able to approve if you're happy with the changes? TYIA 🙂

**Review by jimmy1wu — APPROVED** [review]

**Pull request merged**